lms/views/lti/basic_launch.py

```diff
@@ -259,6 +262,10 @@ def _configure_js_for_file_picker(
         )
 
     def _configure_js_to_show_document(self, document_url, assignment):
+        self.context.js_config.add_document_url(document_url)
+        self.context.js_config.enable_lti_launch_mode(self.course, assignment)
+
+    def _configure_grading(self, document_url, assignment):
```

Comment:
Grading is Canvas vs the rest.

This probably should be "productized" but I reckon it's better to do it in two steps, refactor it so it's all in one place like I'm trying to do here and then move it to the product.

Reply:
If we do the "productization" I reckon "_configure_grading" and "_configure_grading_bar" will merge into one method.